src/ApprenticeAan/SFA.DAS.ApprenticeAan.Api/Program.cs

```diff
@@ -42,10 +44,6 @@
 
 // Configure the HTTP request pipeline.
 
-if (app.Environment.IsDevelopment())
```

Comment:
Was this removed on purpose?

Reply:
Yes, SONAR flagged this as a security hotspot

Reply:
That cannot be right, we do need to see exception page when debugging.

Reply:
SONAR cloud doesn't keep the logs, so this historical run is the most detailed report I can get you, but it did in fact flag it.

![image](https://user-images.githubusercontent.com/28269309/217803685-af0b3511-1006-4494-8ad7-55719db9ef38.png)


Not sure what's changed with it now, but I've reinstated it.